src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -239,6 +243,10 @@ void GlobalAppSettings::LayerJson(const Json::Value& json)
     {
         _initialCols = initialCols.asInt();
     }
+    if (auto rowsToScroll{ json[JsonKey(RowsToScrollKey)] })
+    {
+        _rowsToScroll = rowsToScroll.asInt();

```

Comment:
If we did want to make this either `"system"` or _an int_, we could have something like the following here:
```c++
if (rowsToScroll.isInt())
{
    _rowsToScroll = rowsToScroll.asInt();
}
else if (rowsToScroll.isString() && rowsToScroll.asString() == "system")
{
    _rowsToScroll = DoTheSystemLookupThing();
}
```

Reply:
OK I've updated it so that you can use "system" for a default. In reality, anything that is not a non-zero int will now cause the system default to be used. Makes it a bit more robust.